From 7c69b2a1ae172e64ac836ea62634a5d70ac480dc Mon Sep 17 00:00:00 2001
From: Bruno Ernesto Russo <root@synex.ar>
Date: Sat, 13 Jun 2026 22:47:20 -0300
Subject: [PATCH 1/2] Synex: replace pop-fonts with fonts-inter (not available
 in Debian)

---
 debian/control | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control b/debian/control
index 3fa9fb1..9568ad5 100644
--- a/debian/control
+++ b/debian/control
@@ -36,7 +36,7 @@ Depends:
   fonts-open-sans,
   gnome-keyring,
   libsecret-1-0,
-  pop-fonts,
+  fonts-inter,
   switcheroo-control,
   xdg-desktop-portal-cosmic,
   xwayland,
-- 
2.47.3


From 08f43c9b0acd6ebfd96f152c90990f4368cd2aec Mon Sep 17 00:00:00 2001
From: Bruno Ernesto Russo <root@synex.ar>
Date: Sun, 14 Jun 2026 22:12:43 -0300
Subject: [PATCH 2/2] Synex: add Debian changelog entry

---
 debian/changelog | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/debian/changelog b/debian/changelog
index e890468..5d81b96 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,9 @@
+cosmic-session (1.0.0+synex1) UNRELEASED; urgency=medium
+
+  * Replace pop-fonts with fonts-inter (not available in Debian)
+
+ -- Synex Team <root@synex.ar>  Sun, 14 Jun 2026 01:52:31 -0300
+
 cosmic-session (1.0.0) noble; urgency=medium
 
   * Initial release
-- 
2.47.3

